lib/rucio/core/transfer.py

```diff
@@ -559,16 +559,15 @@ def get_transfer_requests_and_source_replicas(process=None, total_processes=None
                 allow_tape_source = True
 
                 # Find matching scheme between destination and source
-                # TODO: In the future, move to domain "third_party_copy"
                 try:
                     matching_scheme = rsemgr.find_matching_scheme(rse_settings_dest=rses_info[dest_rse_id],
                                                                   rse_settings_src=rses_info[source_rse_id],
                                                                   operation_src='read',
```

Comment:
I was under the impression we do third_part_copy to third_party_copy, thus also using this as the source operation?

Reply:
we should, that's why i asked ;-)